### ISSUE TITLE
impaired neuronal physiology

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3416]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

impaired neuronal physiology 

>2. Synonyms (e.g., absent spleen)

neuronal dysfunction

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

any anomaly resulting in the reduced function of a neuron

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://doi.org/10.1016/S0091-3057(96)00420-0

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal neuron physiology 

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
MP:0021001

It would be helpful to have the PubMed ID for the definition source.

Anna, I added Sarah's ORCID as the "contributor" in the MP-edit file
